### DISCUSSION TITLE
Não instala no PHP 8.1

### DISCUSSION THREAD

**Discussion by ederbatera** [RamonSilva20/mapos#1887] [General]
Tentei instalar no Debian 11 e no Windows/Xampp.
Ambos com erro:
![image](https://user-images.githubusercontent.com/53010526/182979492-c8050aab-b072-4716-b692-0fd587f8d86b.png)

**Comment by Peffmb** [comment]
Estou com o mesmo problema, alguma solução?

**Reply by Jhenriquek1** [reply]
consegui desta forma
https://www.youtube.com/watch?v=qpHXdM_x-Lc

**Reply by brcz779** [reply]
envia o link atualizado do video

**Comment by gildomatos** [comment]
Você tem extensões do PHP desabilitadas, você precisa ir até o arquivo do PHP.ini e habilitar essas extensões.

**Comment by willph** [comment]
Projeto com suporte ao php8 esta em desenvolvimento e pode ser testado baixando no site [mapos.com.br](https://mapos.com.br) no menu superior Download(Beta)

**Comment by andremp15** [comment]
Também estou com o mesmo problema não consegui resolver .

**Comment by willph** [comment]
Está em fase de desenvolvimento e teste. Em breve estará liberado de forma
oficial.

Em qui, 13 de abr de 2023 22:33, andremp15 ***@***.***>
escreveu:

**Comment by willph** [comment]
Já está liberado a nova versão do MapOS com suporte ao php 8.1, só baixar e instalar.

**Reply by sigmaonbr** [reply]
Olá, Onde pode ser baixado, no site, os dois links estão desativados e não achei aqui no github,

**Comment by willph** [comment]
Pode ser no site ou pelo github.com pelo link
https://github.com/RamonSilva20/mapos/releases


No site o único link desativado é a versão beta.

Em seg, 15 de mai de 2023 16:11, WilsonJR ***@***.***>
escreveu:
